### PR TITLE
Change BGP-triggered GUE test to Exchange eBGP route                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Instead of using a gRIBI route                                                                                                                                                                                                                                                                        

### PR DESCRIPTION
**Pull request by wenovus**
Instead of using a gRIBI route

### CODE REVIEW

File: internal/packetutil/capture.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	}
+	defer f.Close()
+
+	handleRead, err := pcapgo.NewReader(f)
+	if err != nil {
+		return fmt.Errorf("could not create reader on pcap file %s: %v", f.Name(), err)
+	}
+	ps := gopacket.NewPacketSource(handleRead, layers.LinkTypeEthernet)
+
+	for i := 0; i != 10; i++ {
```
Comment (DanG100) [inline internal/packetutil/capture.go:41]: why only 10 packets?  maybe make this an argument

File: internal/packetutil/capture.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		return fmt.Errorf("packetutil: zero captured bytes")
+	}
+
+	handleRead, err := pcapgo.NewReader(bytes.NewReader(captureBytes))
+	if err != nil {
+		return fmt.Errorf("could not create reader on captured bytes")
+	}
+	ps := gopacket.NewPacketSource(handleRead, layers.LinkTypeEthernet)
+
+	for i := 0; i != 10; i++ {
```
Comment (DanG100) [inline internal/packetutil/capture.go:62]: same here

File: internal/packetutil/capture.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcapgo"
+)
+
+func DisplayCaptureFile(pcapfile string) error {
```
Comment (DanG100) [inline internal/packetutil/capture.go:28]: func comment

File: internal/packetutil/capture.go [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+		pkt, err := ps.NextPacket()
+		if err != nil {
+			return fmt.Errorf("error reading next packet: %v", err)
+		}
+		fmt.Println(pkt.Dump())
+	}
+	return nil
+}
+
+func DisplayCapture(captureBytes []byte) error {
```
Comment (DanG100) [inline internal/packetutil/capture.go:51]: func comment

File: integration_tests/twodut_oneotg_tests/bgp_triggered_gue/bgp_triggered_gue_test.go [hunk 5]
```diff
@@ -150,38 +155,74 @@ var (
 	}
 )
 
-// configureATE configures port1 and port2 on the ATE.
-func configureATE(t *testing.T, ate *ondatra.ATEDevice) gosnappi.Config {
+// configureOTG configures port1 and port2 on the ATE.
```
Comment (DanG100) [inline integration_tests/twodut_oneotg_tests/bgp_triggered_gue/bgp_triggered_gue_test.go:158]: update comment to include what is being configured. also look like more than 2 ports are configured

File: internal/packetutil/capture.go (hunk 1 above)
Comment (wenovus) [reply internal/packetutil/capture.go]: Made this an argument.

File: internal/packetutil/capture.go (hunk 3 above)
Comment (wenovus) [reply internal/packetutil/capture.go]: Updated.

File: integration_tests/twodut_oneotg_tests/bgp_triggered_gue/bgp_triggered_gue_test.go (hunk 5 above)
Comment (wenovus) [reply integration_tests/twodut_oneotg_tests/bgp_triggered_gue/bgp_triggered_gue_test.go]: I left a general comment here and let the comments within the function talk about more details.

**Review by DanG100 — APPROVED** [review]

**Pull request merged**